### ISSUE TITLE
no pokemon

### ISSUE THREAD

**Issue by Mortalit** [AHAAAAAAA/PokemonGo-Map#3455]
## Steps to Reproduce (for bugs)

2016-08-04 16:55:29,670 [        search] [   INFO] Login to Pokemon Go successful.
2016-08-04 16:55:30,095 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:55:36,058 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:55:41,474 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:55:46,868 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:55:52,289 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:55:57,686 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:56:03,641 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:56:09,035 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:56:14,466 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:56:19,844 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-04 16:56:25,710 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms

## Your Environment
* Version used:
* Environment name and version (e.g. Python 2.7):Python 2.7
* Operating System and version (desktop or mobile): win7

**Comment by martixingwei** [comment]
README.md:
> API is currently down, please don't submit issues. Actively working on fix, might be a while.

**Comment by Sharke** [comment]
Strange I've _never_ seen this issue before.

**Comment by smontana** [comment]
lol

**Comment by Invaderofbunnie** [comment]
[https://github.com/AHAAAAAAA/PokemonGo-Map/issues/3406](url)

This issue hasn't been posted to death yet -_- why do people not search issues before posting!

**Comment by Invaderofbunnie** [comment]
#3406

**Comment by Sydcomebak** [comment]
Hey, is something wrong with the servers or something?  My map isn't working...

</sarcasm>

**Comment by Chippi** [comment]
![image](https://cloud.githubusercontent.com/assets/6388933/17400318/7470c7c4-5a47-11e6-93e8-beea33af47de.png)

**Comment by MikeDX** [comment]
@Invaderofbunnie They never did before, so why should they start now? :)

**Comment by forsakenzombi** [comment]
in some defense of posts like these. I came in this morning and was buzzing around everywhere to see if anyone discovered a fix for this as of yet (I learned about the issue yesterday when it first started happening) and its not exactly easy to seek out that information on this website. There are so many threads reporting this issue and trying to read all of them looking for signs that someone resolved wasnt very efficient. But I assume that it hasnt yet due to this thread. I see both scripts were updated like 4 or 5 hours ago which is confusing cause I immediately thought it was a fix. :-/
